runtime(doc): Add more hyperlinks to :help builtin.txt

dkearns opened the pull request:
Markup remaining builtin function references in descriptions as hyperlinks rather than Ex commands.

Edit: Ex command hyperlinking removed, as discussed below.

zeertzjq commented:
IIRC there was a previous runtime update (over 2 years ago) doing the reverse of this change, but I can't find it now.

dkearns commented:
Hmmm, thanks.  Looks like it might be 1588bc8?

It seems to be limited to Ex commands and mostly replaces generic quoted commands with concealed backtick markup.  I assume they're supposed to have their own highlighting like options.

dkearns committed:
runtime(doc): Add more hyperlinks to :help builtin.txt

Markup remaining builtin function references in descriptions as
hyperlinks rather than Ex commands.

Signed-off-by: Doug Kearns <dougkearns@gmail.com>
